### ISSUE TITLE
Flow collections always wrap at >80 characters, regardless of lineWidth option

### ISSUE THREAD

**Issue by TheBITLINK** [eemeli/yaml#507]
**Describe the bug**
When using the `flow` collection style, collections always seem to wrap at around 80 characters, regardless of the `lineWidth` option:

```yaml
blockCollection:
  aKey: hello
  anotherKey: world
  yetAnotherKey: "!!!"
listOfFlowCollection:
  - {
      aKey: hello,
      anotherKey: world,
      yetAnotherKey: "!!!",
      oneMoreKey: "!!!!!!!!"
    }
  - {
      aKey: hello,
      anotherKey: world,
      yetAnotherKey: "!!!",
      oneMoreKey: "!!!!!!!!"
    }
flowColTwo:
  - { aKey: hello, bKey: world, cKey: "!!!" }
  - { aKey: hello, bKey: world, cKey: "!!!" }
  - { aKey: hello, bKey: world, cKey: "!!!", dKey: "!" }
```

**To Reproduce**
```js
  function serialize() {
    const doc = new YAML.Document({
      blockCollection: {
        aKey: 'hello',
        anotherKey: 'world',
        yetAnotherKey: '!!!'
      },
      listOfFlowCollection: [
        { aKey: 'hello', anotherKey: 'world', yetAnotherKey: '!!!', oneMoreKey: '!!!!!!!!'  },
        { aKey: 'hello', anotherKey: 'world', yetAnotherKey: '!!!', oneMoreKey: '!!!!!!!!'  },
      ],
      flowColTwo: [
        { aKey: 'hello', bKey: 'world', cKey: '!!!'  },
        { aKey: 'hello', bKey: 'world', cKey: '!!!'  },
        { aKey: 'hello', bKey: 'world', cKey: '!!!',  dKey: '!' },
      ]
    })
    YAML.visit(doc, {
      Pair(_, pair: any) {
        if (pair.key?.value === 'listOfFlowCollection' || pair.key?.value === 'flowColTwo')  {
          for (const i of pair.value.items) {
            i.flow = true
          }
        }
      }
    })
    return doc.toString({ lineWidth: 120 })
  }
```

**Expected behaviour**
Flow collections respect the `lineWidth` option and wrap at 120 characters instead of 80 (or not wrap at all if `lineWidth` is 0)

```yaml
blockCollection:
  aKey: hello
  anotherKey: world
  yetAnotherKey: "!!!"
listOfFlowCollection:
  - { aKey: hello, anotherKey: world, yetAnotherKey: "!!!", oneMoreKey: "!!!!!!!!" }
  - { aKey: hello, anotherKey: world, yetAnotherKey: "!!!", oneMoreKey: "!!!!!!!!" }
flowColTwo:
  - { aKey: hello, bKey: world, cKey: "!!!" }
  - { aKey: hello, bKey: world, cKey: "!!!" }
  - { aKey: hello, bKey: world, cKey: "!!!", dKey: "!" }
```

**Versions (please complete the following information):**

- Environment: Chrome 119.0.6045.159
- `yaml`: 2.3.4

**Additional context**
The same issue manifests when using `collectionStyle: 'flow'` in the `toString()` options, so it's not specific to mixed style.

**Comment by ben519** [comment]
Funny enough, I just stumbled my way here too. Fully expected `doc.toString({ lineWidth: 120 })` to set the line width after reading [the docs](https://eemeli.org/yaml/#tostring-options).

**Comment by eemeli** [comment]
A PR making `lineWidth` be used for flow collections instead of the current undocumented approach would be welcome.

**Comment by eemeli** [comment]
Fixed by #522.
